fees/iotex/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Adapter, FetchOptions, ProtocolType } from "../../adapters/types";
+import { getTimestampAtStartOfDayUTC } from "../../utils/date";
+import { httpGet } from "../../utils/fetchURL";
+import { CHAIN } from "../../helpers/chains";
+
+const IOTEX_SUM_GAS_URL = "https://gateway1.iotex.me/analyzer/sumGasFeeIotx";
+
+const fetch = async (options: FetchOptions) => {
+  const dailyFees = await getDailyFees(options.startTimestamp);
+  const totalFees = await getTotalFees();
+
+  return {
+    dailyFees,
+    dailyRevenue: dailyFees,
```

Comment:
just checked, we treat only the fees burnt as revenue for chains: https://github.com/DefiLlama/dimension-adapters/blob/master/fees/solana.ts#L20

What percentage of fees is burnt in iotex?

Reply:
Hi @g1nt0ki,

Thanks for your feedback! To clarify, in the IoTeX ecosystem, the fees are sent to the rewarding pool managed entirely by the protocol. This pool, located at the special address [io0000000000000000000000rewardingprotocol](https://iotexscan.io/address/io0000000000000000000000rewardingprotocol).

Importantly, these tokens remain under the protocol’s control and are only accessible to delegates through the client command `ioctl action claim`. This mechanism ensures that the fees are directed back into the ecosystem, effectively acting as revenue that the protocol manages internally for the benefit of its stakeholders.

According to [DefiLlama’s documentation](https://docs.llama.fi/list-your-project/other-dashboards/dimensions), `dailyRevenue` _includes fees kept by the protocol and distributed to token holders, developers, or the treasury_. Since IoTeX fees fulfill this definition by being allocated to delegates and in turn to stake holders, shouldn't they qualify as protocol revenue?

Reply:
by delagates, you mean the stakers? if that is the case, then no, it is not revenue

Reply:
ah, sorry no, we cant count it as revenue

Reply:
also, what is counted as fees here? what users pay or from chain inflation?

Reply:
@g1nt0ki removed revenue dimension

Reply:
> also, what is counted as fees here? what users pay or from chain inflation?

Its gas fees paid by users, similar to other chains